packages/core-mobile/app/navigation/wallet/NFTSendStack.tsx

```diff
@@ -68,3 +74,28 @@ const NftSendScreen = (): JSX.Element => {
     />
   )
 }
+
+// TODO: after migrating vm-module's nft balance fetching, use the real values from the nft balance
+const mapTokenFromNFT = (nft: NFTItemData): NftTokenWithBalance => {
+  return {
+    tokenId: nft.tokenId,
+    type: isErc721(nft) ? TokenType.ERC721 : TokenType.ERC1155,
+    address: nft.address,
+    logoUri: nft.metadata.imageUri ?? '',
+    name: nft.metadata.name ?? '',
+    symbol: isErc721(nft) ? nft.symbol : '',
+    //unused but included to conform to NftTokenWithBalance
+    balanceInCurrency: 0,
+    balanceDisplayValue: '',
+    balanceCurrencyDisplayValue: '',
+    priceInCurrency: 0,
+    description: '',
+    marketCap: 0,
+    change24: 0,
+    vol24: 0,
+    balance: 0n,
+    logoSmall: '',
+    collectionName: isErc721(nft) ? nft.name : nft.metadata.name ?? 'Unknown',
+    tokenUri: ''
+  }
```

Comment:
In the future, the balance of NFT tokens should be retrieved from the VM module.